src/components/AnimatedStep/AnimatedStepContext.ts

```diff
@@ -0,0 +1,15 @@
+import React, {createContext} from 'react';
+import {ValueOf} from 'type-fest';
+import CONST from '../../CONST';
+
+type AnimationDirection = ValueOf<typeof CONST.ANIMATION_DIRECTION>;
+
+type StepContext = {
+    animationDirection: AnimationDirection;
+    setAnimationDirection: React.Dispatch<React.SetStateAction<ValueOf<typeof CONST.ANIMATION_DIRECTION>>>;
```

Comment:
```suggestion
    setAnimationDirection: Dispatch<SetStateAction<AnimationDirection>>;
```

Reply:
Right, that's why I created it 😅 Updated!